sqlmesh/dbt/model.py

```diff
@@ -20,6 +21,11 @@
 from sqlmesh.dbt.target import TargetConfig
 from sqlmesh.utils.errors import ConfigError
 
+if sys.version_info >= (3, 9):
```

Comment:
Should this be removed?

Reply:
Surprised these weren't auto-removed when the dependencies were removed.